includes/Products.php

```diff
@@ -1087,6 +1087,10 @@ public static function get_product_pattern( \WC_Product $product ) {
 	 * @return \WC_Product_Attribute[]
 	 */
 	public static function get_available_product_attributes( \WC_Product $product ) {
+		if ( $product->is_type( 'variation' ) ) {
+			$parent_product = wc_get_product( $product->get_parent_id() );
+			return $parent_product instanceof \WC_Product ? self::get_available_product_attributes( $product ) : array();
```

Comment:
I think you meant to use `$parent_product` here:

````suggestion
			return $parent_product instanceof \WC_Product ? self::get_available_product_attributes( $parent_product ) : array();
```

Reply:
Oops, good catch and thank you for the test. Lazy of me to not add one myself.